spec/element_spec.rb

```diff
@@ -45,6 +45,23 @@ class PageWithAFewElements < SitePrism::Page
     expect(page).to respond_to :all_there?
   end
 
+  it 'checks only expected_elements if excluded_elements specified' do
+    class PageWithAFewElements < SitePrism::Page
+      element :bob, 'a.b c.d'
```

Comment:
Same as description on README, up to you though as I know it's in keeping style.